README.md

```diff
@@ -7,37 +7,34 @@
 > The **vital** circulating fluid of a plant.
 
 
-`linfa` aims to provide a comprehensive toolkit to build Machine Learning applications
-with Rust.
+`linfa` aims to provide a comprehensive toolkit to build Machine Learning applications with Rust.
 
-Kin in spirit to Python's `scikit-learn`, it focuses on common preprocessing tasks
-and classical ML algorithms for your everyday ML tasks.
+Kin in spirit to Python's `scikit-learn`, it focuses on common preprocessing tasks and classical ML algorithms for your everyday ML tasks.
 
 _Documentation_: [latest](https://docs.rs/linfa)
-_Community chat_: [Gitter](https://gitter.im/linfa-ml/community)
+_Community chat_: [Zulip](https://rust-ml.zulipchat.com/)
 
 ## Current state
 
 Such bold ambitions! Where are we now? [Are we learning yet?](http://www.arewelearningyet.com/)
```

Comment:
I would personally re-word this to more neutral language, but its obviously a matter of opinion.

Reply:
Moved away from "such bold ambitions" in latest commit